### ISSUE TITLE
Crash when deleting incomplete path

### ISSUE THREAD

**Issue by madig** [trufont/trufont#153]
1. Construct a path with 2 points
2. Ctrl-A and delete them
3. Repeat 1
4. Repeat 2

```
Traceback (most recent call last):
  File "/home/nikolaus/Entwicklung/trufont/Lib/defconQt/glyphView.py", line 1331, in keyPressEvent
    item.delete(not event.modifiers() & Qt.ShiftModifier)
  File "/home/nikolaus/Entwicklung/trufont/Lib/defconQt/glyphView.py", line 815, in delete
    self.getSegmentIndex(), preserveShape)
  File "/home/nikolaus/.local/lib/python3.4/site-packages/defcon/objects/contour.py", line 385, in removeSegment
    raise NotImplementedError("unknown segment type: %s" % nextSegment[-1].segmentType)
NotImplementedError: unknown segment type: move
```

**Comment by adrientetar** [comment]
Thanks. This will be fixed as part of #150 (5bcda408246e6775efe9997c4650bcfa5882bb96).

**Comment by adrientetar** [comment]
Now merged.
